mcdc/type_.py

```diff
@@ -25,9 +30,96 @@
 setting = None
 tally = None
 technique = None
+translate = None
+group_array = None
+j_array = None
 global_ = None
 
 
+# ==============================================================================
+# Alignment Logic
+# ==============================================================================
+
```

Comment:
The explanation of the need for alignment in the PR Conversation is excellent. Let's put it here as well.

Reply:
🫡